doc/source/user_guide/10min.rst

```diff
@@ -16,6 +16,15 @@ Customarily, we import as follows:
    import numpy as np
    import pandas as pd
 
+Basic data structures in pandas
+------------------------------
+
+Information in pandas is organized in two types of classes:
+
+1. :class:`Series`: a one-dimensional container of indexed data. It is analogous to arrays and dictionaries.
+2. :class:`DataFrame`: a two-dimensional container of indexed data.  \
+   It follows a tabular data format, where rows (indexes) and columns are used to organize information.
+
```

Comment:
If I'm not wrong, we should have page in our docs about the pandas data structures. It'd be good to link it here for anyone that needs further information.